packages/stacks-classic/lib/components/pagination/pagination.less

```
@@ -9,69 +9,62 @@
 
     & &--item {
         --_pa-item-bg: transparent;
-        --_pa-item-bc: var(--bc-darker);
-        --_pa-item-fc: var(--fc-medium);
-        --_pa-item-bg-focus: var(--black-400);
-        --_pa-item-fc-focus: var(--white);
-        --_pa-item-bg-hover: var(--black-225);
-        --_pa-item-bc-hover: var(--bc-darker);
-        --_pa-item-fc-hover: var(--fc-dark);
-
-        // CONTEXTUAL STYLES
-        .highcontrast-mode({ text-decoration: none; });
+        --_pa-item-br: unset;
+        --_pa-item-bs: none;
+        --_pa-item-fc: var(--black-400);
+        --_pa-item-p: var(--su-static4);
 
         // MODIFIERS
         &.is-selected {
-            --_pa-item-bg: var(--theme-primary);
-            --_pa-item-bc: transparent;
-            --_pa-item-fc: var(--white);
-            --_pa-item-bg-focus: var(--theme-primary-400);
-        }
-        &.s-pagination--item__clear {
-            --_pa-item-bg: transparent;
-            --_pa-item-bc: transparent;
-            --_pa-item-fc: inherit;
+            --_pa-item-bs: 0 var(--su-static2) 0 0 var(--black-600);
+            --_pa-item-fc: var(--black-600);
         }
-        // override hover styles to match base styles
-        &.is-selected,
-        &.s-pagination--item__clear {
-            --_pa-item-bc-hover: var(--_pa-item-bc);
-            --_pa-item-bg-hover: var(--_pa-item-bg);
-            --_pa-item-fc-hover: var(--_pa-item-fc);
+        &.s-pagination--item__nav {
+            --_pa-item-bg: unset;
```

I think this is because of https://github.com/StackExchange/Stacks/pull/2035#discussion_r2501028120

> I'd set this as a psuedo private custom property with a value of unset at the top of the file and reset the value here to prevent cascade issues.


I was intending the base background color (line 11) to be set to `unset` and I now realize saying "reset the value here" is unclear. Sorry about that!

For the nav items, `--_pa-item-bg` should be set to `--black-150`.

```suggestion
            --_pa-item-bg: var(--black-150);
```